docs/src/_theme/DocSidebar/index.jsx

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useCallback, useEffect, useRef } from 'react';
```

Comment:
is this file generated by sizzling? if so, should we mention it somewhere in the docstring of the file?